client/src/components/AnswerInfoPopup/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import Button from '@material-ui/core/Button';
```

Comment:
You can use one import statement for files to be imported from the same module.
E.g. import { Button, Dialog } from '@material-ui/core'